applications/openshift/networking/configure_network_policies_hypershift_hosted/rule.yml

```diff
@@ -0,0 +1,62 @@
+documentation_complete: true
+
+prodtype: ocp4
+
+platform: ocp4-on-hypershift
```

Comment:
Per the [style guide](https://complianceascode.readthedocs.io/en/latest/manual/developer/04_style_guide.html#rule-sections) please ensure that your section are in the correct order.

Reply:
thanks for the review, I will fix that